<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PictureCardInfo</class>
 <widget class="QWidget" name="PictureCardInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>166</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>379</width>
    <height>166</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QLabel" name="pictureLabel">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>167</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../resources/qrcs/addPerson.qrc">:/images/images/Person.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="sourceLabel">
         <property name="font">
          <font>
           <family>Inter</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Source</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="dateLabel">
         <property name="font">
          <font>
           <family>Inter</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sizeLabel">
         <property name="font">
          <font>
           <family>Inter</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Size</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="sourceInfo">
         <property name="font">
          <font>
           <family>Inter</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Unkown</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="dateInfo">
         <property name="font">
          <font>
           <family>Inter</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Unkown</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sizeInfo">
         <property name="font">
          <font>
           <family>Inter</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Unkown</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/qrcs/addPerson.qrc"/>
 </resources>
 <connections/>
</ui>
